config/locales/en.yml

```diff
@@ -218,6 +218,12 @@ en:
   authorize_follow:
     error: Unfortunately, there was an error looking up the remote account
     follow: Follow
+    following_html: 'Success! You (<strong>%{self}</strong>) are now following:'
```

Comment:
It is probably not necessary to specify self in these two strings because the user just clicked from a page that already did that.